src/main/java/com/pnu/ordermanagementapp/order/OrderService.java

```diff
@@ -0,0 +1,17 @@
+package com.pnu.ordermanagementapp.order;
+
+import com.pnu.ordermanagementapp.model.Order;
+import org.springframework.data.domain.Page;
+
+public interface OrderService {
+
+    Page<Order> findByClientId(Long clintId, int pageNumber, Long userId);
+
+    Page<Order> findByProductId(Long productId, int pageNumber, Long userId);
+
+    Page<Order> findAll(int pageNumber, Long userId);
```

Comment:
Why don't you pass a `pageable argument` directly? What the goal of `pageNumber`?

Reply:
We receive `pageNumber` as query parameter from UI and build pagable in service